js/views/participantview.js

```diff
@@ -139,7 +139,7 @@
 					return '<span><div class="avatar" data-user="' + escapeHTML(element.id) + '" data-user-display-name="' + escapeHTML(element.displayName) + '"></div>' + escapeHTML(element.displayName) + '</span>';
 				},
 				formatSelection: function () {
-					return '<span class="select2-default" style="padding-left: 0;">'+OC.L10N.translate('spreed', 'Choose person…')+'</span>';
+					return '<span class="select2-default" style="padding-left: 0;">'+OC.L10N.translate('spreed', 'New call …')+'</span>';
```

Comment:
This is wrong, it adds a user to the participant list.
Anyway the naming is fixed in https://github.com/nextcloud/spreed/pull/751

Should not be done in a PR that deals with "sidebar toggle"